client/my-sites/stats/stats-subscribers-highlight-section/index.tsx

```diff
@@ -102,11 +106,28 @@ function SubscriberHighlightsListing( { siteId }: { siteId: number | null } ) {
 	);
 }
 
+function SubscriberLaunchpadSection( { siteId }: { siteId: number | null } ) {
+	const { data: subscribersTotals, isLoading } = useSubscribersTotalsQueries( siteId );
+
+	const isSimple = useSelector( isSimpleSite );
+	const isAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
+	const locale = useLocale();
+
+	const showLaunchpad =
+		! isLoading &&
+		( isSimple || isAtomic ) &&
+		SubscriberLaunchpad.hasTranslationsAvailable( locale ) &&
```

Comment:
Is this translation check still needed? Looking at the status of those translations (https://translate.wordpress.com/developer/?path=client/my-sites/subscribers/components/subscriber-launchpad/subscriber-launchpad.tsx), all languages seems to be in already.

This also means we could consider removing the whole `SubscriberLaunchpad.hasTranslationsAvailable`.

Reply:
Done. :)

Reply:
Thanks! 🙂 Reviewing again.